Runtime/Scripts/DebugUtils/RGGizmos.cs

```diff
@@ -289,7 +291,7 @@ public void DestroyAllSpheres()
          * Draws all Gizmos that have been set.
          * </summary>
          */
-        protected internal void OnDrawGizmos()
+        public void OnDrawGizmos()
```

Comment:
I'm not thrilled with this change, but it was the cleanest backwards compatible solution I found for being able to call this internally from our SDK classes, not just user bots.